config/routes.js

```diff
@@ -519,6 +526,80 @@ module.exports.routes = {
     action: 'saveAppConfig',
     csrf: false
   },
+  // i18next http-backend compatible route to fetch namespaces
+  // Example: /default/rdmp/locales/en/translation.json
+  'get /:branding/:portal/locales/:lng/:ns.json': {
```

Comment:
From what I understand, the `/:branding/:portal/locales*` routes don't have entries in `auth.js`. Is that intended?